plugins/pagerduty/src/components/PagerDutyHomepageCard/Content.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+
+import { PagerDutyEntity } from '../../types';
+import { PagerDutyCard } from '../PagerDutyCard';
+
+/** @public */
+export type PagerDutyHomepageCardProps = PagerDutyEntity & {
+  readOnly?: boolean;
+};
+
+/** @public */
+export const Content = (props: PagerDutyHomepageCardProps) => {
```

Comment:
what's this for?

Reply:
I believe the home page interface requires an object with a key "Component" containing the component that you want to display on the home page. I think I am just following a pattern I saw for other home page components:

https://github.com/backstage/backstage/blob/master/plugins/home/src/homePageComponents/RandomJoke/Content.tsx#L21